docs/source/index.rst

```diff
@@ -15,6 +15,13 @@ JWST
    jwst/datamodels/index.rst
    jwst/transforms/index.rst
 
+AsdfInFits
+==========
+
+.. toctree::
+   :maxdepth: 1
+
+   asdf_in_fits.rst
```

Comment:
This should be moved up to the first `toctree` on line 4.

Reply:
I moved asdf_in_fits.rst to the other toctree and added a title and link.
https://stdatamodels--114.org.readthedocs.build/en/114/
Let me know if these were not the changes you were requesting.